lib/modules/quickMessage.js

```diff
@@ -152,33 +152,26 @@ addModule('quickMessage', function(module, moduleID) {
 			RESStorage.setItem('RESmodules.quickMessage.lastSentAs.' + RESUtils.loggedInUser(), this.value);
 		});
 
-		var subreddit = RESUtils.currentSubreddit(),
-			messageTheMods = document.querySelector('.side a.helplink');
-		if (module.options.quickModeratorMessage.value && subreddit && messageTheMods) {
-			messageTheMods.addEventListener('click', function(e) {
-				if (e.which === 1) {
-					e.preventDefault();
-					module.openQuickMessageDialog({'to': '/r/' + subreddit});
-				}
-			});
-		}
+		function messageLinkEventHandler(event) {
+			if (event.which === 1) {
+				event.preventDefault();
 
-		if (module.options.openInlineLinks.value) {
-			$('div.content').on('click', 'a[href*="/message/compose"]', function(e) {
```

Comment:
It's probably worth sanity checking that the link either exactly starts with `/message/compose` or matches `^https?://blah.reddit.com/message/compose`